src/applications/virtual-agent/tests/components/webchat/startConvoAndTrackUtterances.unit.spec.js

```diff
@@ -283,7 +283,7 @@ describe('makeBotStartConvoAndTrackUtterances actions', () => {
       const spyDispatchEvent = sandbox.spy(window, 'dispatchEvent');
       const activity = {
         type: 'event',
-        name: 'RX_Skill_Exit',
+        name: 'Skill_Exit',
```

Comment:
We need to mock the value as well. @nichelous-herndon

Reply:
@ksk385 Added the changes.